core/src/main/java/org/jclouds/proxy/ProxyForURI.java

```diff
@@ -84,9 +81,20 @@ public PasswordAuthentication getPasswordAuthentication() {
             Authenticator.setDefault(authenticator);
          }
          return proxy;
-      } else {
-         return Proxy.NO_PROXY;
       }
+      if (config.canUseProxyFromJvm()) {
+         return getDefaultProxy(endpoint);
+      }
+      if (config.useSystem()) {
+         System.setProperty("java.net.useSystemProxies", "true");
```

Comment:
Add a comment here to explain that this actually has little effect in practice? I.e. the same comment as in the PR explanation?